bin/cron/deploy_to_test

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+
+require 'cdo/chat_client'
+require 'cdo/github'
+require 'cdo/only_one'
+
+# Checks whether the topic of Slack#developers includes "DTT: yes", updating it
+# to reflect auto-DTT if so.
+# @return [Boolean] Whether to proceed with the auto-DTT.
+def check_and_update_developers_topic
+  current_topic = Slack.get_topic('developers')
+  unless current_topic.include? 'DTT: yes'
+    ChatClient.message(
+      'infra-test',
+      "auto-DTT aborted (\#developers topic is #{current_topic}).",
```

Comment:
Maybe "Skipping auto-DTT" here?  We're not aborting it so much as deciding not to start it.